### PR TITLE
Javadoc ndg

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

File: docs/images/investments/watchlistSequence.png
Comment (ryan1604) [inline docs/images/investments/watchlistSequence.png:1]: I think for consistency its better for either all methods to have activations bars or none at all

File: docs/images/vis/categorizerSequence.png
Comment (ryan1604) [inline docs/images/vis/categorizerSequence.png:1]: here too

**Review by ryan1604 — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/investments/WatchList.java [hunk 1]
```diff
@@ -218,10 +236,9 @@ public void extractWatchlistInfoFromJSONArray(JSONArray jsonstocks) throws Finan
      *
      * @param stock
      * @param stockLocal
-     * @param fetchTime
      */
-    public void extractStockInfoFromJSONObject(JSONObject stock, Stock stockLocal, long fetchTime) {
-        stockLocal.setLastFetched(fetchTime);
+    public void extractStockInfoFromJSONObject(JSONObject stock, Stock stockLocal) {
+        //stockLocal.setLastFetched(fetchTime);
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/investments/WatchList.java:241]: delete this if not needed

**Review by NeoMinWei — APPROVED** [review]
LGTM

**Pull request merged**